go.mod

```diff
@@ -7,29 +7,29 @@ require (
 	github.com/Microsoft/go-winio v0.6.1
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/blang/semver/v4 v4.0.0
-	github.com/checkpoint-restore/checkpointctl v0.1.0
-	github.com/checkpoint-restore/go-criu/v6 v6.3.0
-	github.com/container-orchestrated-devices/container-device-interface v0.6.0
+	github.com/checkpoint-restore/checkpointctl v1.1.0
+	github.com/checkpoint-restore/go-criu/v7 v7.0.0
+	github.com/container-orchestrated-devices/container-device-interface v0.6.1
 	github.com/containerd/cgroups v1.1.0
-	github.com/containerd/containerd v1.7.5
+	github.com/containerd/containerd v1.7.10
 	github.com/containerd/cri-containerd v1.19.0
 	github.com/containerd/fifo v1.1.0
-	github.com/containerd/nri v0.3.1-0.20230504231226-94185418e253
+	github.com/containerd/nri v0.5.0
 	github.com/containerd/ttrpc v1.2.2
 	github.com/containerd/typeurl v1.0.3-0.20220422153119-7f6e6d160d67
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.3.0
-	github.com/containers/common v0.55.4
+	github.com/containers/common v0.57.0
```

Comment:
This change means that the following Pull Request is no longer needed:

- https://github.com/cri-o/cri-o/pull/7496